python-sdk/nuscenes/eval/predict/README.md

```diff
@@ -0,0 +1,73 @@
+# nuScenes predicton task
+In this document we present the rules, submission process, and evaluation metrics of the nuScenes prediction task.
+
+## Overview
+- [Introduction](#introduction)
+- [Participation](#participation)
+- [Challenges](#challenges)
+- [Submission rules](#submission-rules)
+- [Results format](#results-format)
+- [Evaluation metrics](#evaluation-metrics)
+
+## Introduction
+The goal of the nuScenes prediction task is to predict the future trajectories of objects in the nuScenes dataset.
+A trajectory is a sequence of x-y locations. For this challenge, the predictions are 6-seconds long and sampled at
+2 hertz.
+
+## Challenges
+To allow users to benchmark the performance of their method against the community, we will host a single leaderboard all-year round.
+Additionally, we organize a number of challenges at leading Computer Vision and Machine Learning conference workshops.
+Users that submit their results during the challenge period are eligible for awards.
+Any user that cannot attend the workshop (direct or via a representative) will be excluded from the challenge, but will still be listed on the leaderboard.
+
+### Workshop on Benchmarking Progress in Autonomous Driving, ICRA 2020
+The first nuScenes prediction challenge will be held at [ICRA 2020](https://www.icra2020.org/).
+The submission period will open in early march and continue until May 28th, 2020.
+Results and winners will be announced at the [Workshop on Benchmarking Progress in Autonomous Driving](http://montrealrobotics.ca/driving-benchmarks/).
+Note that the evaluation server can still be used to benchmark your results after the challenge period.
+
+## Submission rules
+### Prediction-specific rules
+* The user can submit up to 25 proposed future trajectories, called `modes`, for each agent along with a probability the agent follows that proposal. Our metrics (explained below) will measure how well this proposed set of trajectories matches the ground truth.
+* Unlike previous challenges, the user will not submit their submissions to the eval server. Instead, we will run the user's model
+on the test set for them. To make this possible, the user should modify the `do_inference_for_submission` function in the [inference script](https://github.com/nutonomy/nuscenes-devkit/blob/nuscenes-predict-challenge/python-sdk/nuscenes/eval/predict/do_inference.py)
+so that their model is used.
+* Up to two seconds of past history can be used to predict the future trajectory for each agent.
+* Every submission to the challenge must be accompanied by a technical report describing the method in sufficient detail to allow for independent verification.
```

Comment:
Want to expand on this a little bit. Right now, it is quite subjective and vague. We could have details on a separate page that we link to (e.g., do we need code, hyperparams for training, data augmentation techniques, model architecture diagrams, etc).

Reply:
I think we get their code automatically since we run their submission for them so I was thinking it'd be focused on explaining the rationale behind their code and their training procedure.